yarn.lock

```diff
@@ -3838,7 +3838,7 @@ colorette@^1.2.1:
   resolved "https://registry.yarnpkg.com/colorette/-/colorette-1.2.1.tgz#4d0b921325c14faf92633086a536db6e89564b1b"
   integrity sha512-puCDz0CzydiSYOrnXpz/PKd69zRrribezjtE9yd4zvytoRc8+RY/KJPvtPFKZS3E3wP6neGyMe0vOTlHO5L3Pw==
 
-colors@^1.1.2, colors@^1.3.3:
+colors@1.4.0, colors@^1.1.2:
```

Comment:
Oh no, the colors package is the one that broke the build some days ago :fearful: 
@luisfelipefurlan Do you think that this will break the build?

Reply:
this particular version is fine.
I made this change and left this version fixed by removing the "~"